conf/distro/debian-buster-backports.list

```diff
@@ -0,0 +1,2 @@
+deb	http://deb.debian.org/debian buster-backports main
+deb-src	http://deb.debian.org/debian buster-backports main
```

Comment:
should the backports be added into snapshot list?

Reply:
I can add it there as placeholder - reproducing the new npm version from the current snapshot date is not working, obviously. But it should be updated on next freeze, yet.

Reply:
It's OK, we can schedule the modification by next release, currently just a placeholder or even a todo comment is sufficient.

Reply:
Done in the updated version.